File: rust/wikibooks-rust-learning-by-making/ch2/cmptext.py
afile = "./fizzbuzz_python.txt"
bfile = "./fizzbuzz_rust.txt"

with open(afile, "r") as fp:
    astr = fp.read()

with open(bfile, "r") as fp:
    bstr = fp.read()

astr = astr.strip()
bstr = bstr.strip()

if astr == bstr:
    print("ok")
else:
    print("ng")

File: rust/wikibooks-rust-learning-by-making/ch2/cmptext.rs
use std::fs;

fn main() {
  let afile = "./fizzbuzz_python.txt";
  let bfile = "./fizzbuzz_rust.txt";

  let astr = fs::read_to_string(afile).unwrap();
  let bstr = fs::read_to_string(bfile).unwrap();

  let astr = astr.trim();
  let bstr = bstr.trim();

  if astr == bstr {
    println!("ok");
  } else {
    println!("ng");
  }
}